### PR TITLE
C_CPP : Multiple Inheritance in C++

### LINKED ISSUE #8181: C_CPP : Multiple inheritance in C++
### Description

I will submit documentation for multiple inheritance in C++.

### Domain

C/CPP

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by vartikavr**
<hr>

## Description 📜

Added documentation for multiple inheritance in C++.
Fixes #8181 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Review by gunjan-g — CHANGES_REQUESTED** [review]
@vartikavr Use cpp markdown instead of normal one, where you've used cpp codes

File: C_CPP/OOPs in C++/Inheritance and its types or Function overriding/multiple_inheritance_doc.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+Inside Class C
+```
+
+## Constructors and Destructors in multiple inheritance
+
+The constructors of base classes in multiple inheritance are called in the sequence in which they are inherited. The base class declared first during inheritance will invoke its constructor first, the one declared second will invoke after that, and so on. The opposite order will be followed for the destructors.
+
+### Example
+
+```
```
Comment (gunjan-g) [inline C_CPP/OOPs in C++/Inheritance and its types or Function overriding/multiple_inheritance_doc.md:74]:
```suggestion
```cpp
```

**Review by gunjan-g — CHANGES_REQUESTED** [review]
like this

**Comment by vartikavr** [comment]
@gunjan-g, I've done the suggested changes. Kindly review again.

File: C_CPP/OOPs in C++/Inheritance and its types or Function overriding/multiple_inheritance_doc.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+<p align="center"> <img src="https://user-images.githubusercontent.com/61552413/142762283-e4deada2-c86b-433a-9e54-8cb05ff99f31.png" height="350"> </p>
+
+Here, class D will be regarded as a child class or derived class, and classes A, B and C will be regarded as the parent classes or the base classes.
+
+A real-life example of multiple inheritance could be, class Bat inheriting its properties and characteristics from classes Mammal and wingedAnimal.
+
+## Example
+
+```cpp
+#include <iostream>
```
Comment (apeksha235) [inline C_CPP/OOPs in C++/Inheritance and its types or Function overriding/multiple_inheritance_doc.md:18]: @vartikavr this is a pretty basic example, give another better example along with this.

**Review by apeksha235 — CHANGES_REQUESTED** [review]

**Comment by vartikavr** [comment]
@apeksha235, I've changed and tried to include a better example. Kindly review again.

**Review by apeksha235 — APPROVED** [review]
@Saviour1001 Please merge this

**Pull request merged**